doc/Overview/Localization/LocalizationOverview.md

```diff
@@ -1,46 +1,70 @@
 
-For more documentation on localization, read the references listed at the bottom.
+It is often necessary to adapt an application to a specific subset of users within a market. **Localization** includes the type of actions developers take to modify both user interface elements and content to adhere to more languages or cultures. Specifically, text **translation** is done by applying alternate strings of text at runtime which accomodate a user's language preference.
 
-## Text localization
+Many apps store these pieces of text in dedicated resource files that the app parses and assigns as text throughout the application. `Uno.Extensions.Localization` provides a consistent way to resolve text of a specific culture or locale across platforms. This feature allows for modifying them to be applied upon app restart.
 
-Save locale specific resources in `.resw` files in folder corresponding to locale (eg en-US).
+It uses [Microsoft.Extensions.Localization](https://www.nuget.org/packages/Microsoft.Extensions.Localization) for any localization related work. For documentation on the broader process of localization, read the references listed at the bottom.
 
-An implementation of `IStringLocalizer` (`ResourceLoaderStringLocalizer`) is registered as service.
+## Set up localization
 
 ```csharp
-private IHost Host { get; }
-
-public App()
+protected override void OnLaunched(LaunchActivatedEventArgs e)
 {
-    Host = UnoHost
-        .CreateDefaultBuilder()
-        .UseLocalization()
-        .Build();
-    // ........ //
-}
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => 
+        {
+            host.UseLocalization()
+        });
+...
+```
+
+An implementation of `IStringLocalizer` (`ResourceLoaderStringLocalizer`) will be registered as a service. This service offers a consistent way to resolve localized strings. Behind the scenes, it will automatically use `ResourceManager` on Windows and `ResourceLoader` on other platforms.
+
+### Adding language specific resources
+
+The `ResourceLoaderStringLocalizer` will look for `.resw` files in folders corresponding to the well-known language tag (eg en-US). For example, if the current culture is `en-US`, the `ResourceLoaderStringLocalizer` will look for `.resw` files in the `en-US` folder. If the current culture is `fr-FR`, the `ResourceLoaderStringLocalizer` will look for `.resw` files in the `fr-FR` folder.
+
+To add a new resource file, right click on the project and select **Add > New Item...**. Select **Resource File (.resw)** and name it according to the language tag (eg `en-US.resw`). Resource files have a key-value pair structure. The key is used to identify the resource and the value is the localized text. The key contains a name which corresponds to the `x:Uid` property of the element in XAML. The value contains the localized text.
+
```

Comment:
Start a new paragraph for discussing x:Uid - this is relevant for using resources in XAML. The key in the resources file is made up of 
[x:Uid].[PropertyName] 
This is important as you can specify more than just strings in the resource files eg MyWidget.Width would allow you to change the width of an item based on locale